src/compiler-team.md

```diff
@@ -17,6 +17,13 @@ home. In any case, you can find people in one of three places at the moment:
 - The `t-compiler` stream on [the Zulip instance](https://rust-lang.zulipchat.com/#narrow/stream/131828-t-compiler)
 - The `compiler` channel on the [rust-lang discord](https://discord.gg/rust-lang)
 
+## Expert map
+
+We maintain a [list of areas of the compiler and who on the team knows a
+bit about that particular area](https://github.com/rust-lang/compiler-team/blob/master/experts/MAP.md).
```

Comment:
This link is a 404 for me.

Reply:
It is intended to be until a compiler-team PR lands.

Reply:
For posterity: https://github.com/rust-lang/compiler-team/pull/40

Reply:
I would modify this paragraph in the same way that I suggested on rust-lang/compiler-team#40, I think -- i..e., to give a better idea of how the list is meant to be used, and invite people to ask questions.